server/chat-commands/core.ts

```diff
@@ -728,15 +741,16 @@ exports.commands = {
 	rank(target, room, user) {
 		if (!target) target = user.name;
 
-		Ladders.visualizeAll(target).then(values => {
+		void Ladders.visualizeAll(target).then(values => {
 			let buffer = `<div class="ladder"><table>`;
 			buffer += Chat.html`<tr><td colspan="8">User: <strong>${target}</strong></td></tr>`;
 
-			let ratings = values.join(``);
+			const ratings = values.join(``);
 			if (!ratings) {
 				buffer += `<tr><td colspan="8"><em>This user has not played any ladder games yet.</em></td></tr>`;
 			} else {
-				buffer += `<tr><th>Format</th><th><abbr title="Elo rating">Elo</abbr></th><th>W</th><th>L</th><th>Total</th>`;
+				const fail = `<tr><th>Format</th><th><abbr title="Elo rating">Elo</abbr></th><th>W</th><th>L</th><th>Total</th>`;
+				buffer += fail;
```

Comment:
```suggestion
				buffer += `<tr><th>Format</th><th><abbr title="Elo rating">Elo</abbr></th><th>W</th><th>L</th><th>Total</th>`;
```

I'm pretty sure our length limit regex handles this correctly; if it doesn't, we'll need to fix it in the regex, not here.